# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/gtk/nsWindow.h
# Commit: 0fa3652858e9
# Full Hash: 0fa3652858e9181ace74c9c72e62e46c58eed85a
# Author: stransky <stransky@redhat.com>
# Date: 2022-07-26 21:44:38
# Regressor Bug: 1777664
# File Overlap Count: 1
# Description:
#   Bug 1777664 [Linux] Replace CleanLayerManagerRecursive() with DestroyLayerManager() as they're the same r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D152686
# ==============================================================================

diff -r 2cbab5d7e2a9 -r 0fa3652858e9 widget/gtk/nsWindow.h
--- a/widget/gtk/nsWindow.h	Tue Jul 26 14:49:02 2022 +0000
+++ b/widget/gtk/nsWindow.h	Tue Jul 26 14:56:46 2022 +0000
@@ -768,8 +768,6 @@
 
   void SetCompositorWidgetDelegate(CompositorWidgetDelegate* delegate) override;
 
-  void CleanLayerManagerRecursive();
-
   int32_t RoundsWidgetCoordinatesTo() override;
 
   void UpdateMozWindowActive();
